CHANGELOG.md

```diff
@@ -1,5 +1,208 @@
 # Changelog
 
+## 5.0.0.rc1
+
+This is a major Teleport release with a focus on new features, functionality, and
+bug fixes. It’s a substantial release and users can review [5.0 closed issues](https://github.com/gravitational/teleport/milestone/39?closed=1)
+on Github for details of all items.
+
+#### New Features
+Teleport 5.0 is a major version and an introduction to Teleport's Unified Access
```

Comment:
The unified access plane is mentioned only once outside of this. I know there isn't a need for substantial documentation, but we should have a short header describing what it is and maybe a link to a blog post on the topic.